pkg/bootstrap/static.go

```diff
@@ -0,0 +1,392 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bootstrap
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"time"
+
+	"github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	"github.com/envoyproxy/go-control-plane/envoy/api/v2/auth"
+	"github.com/envoyproxy/go-control-plane/envoy/api/v2/cluster"
+	"github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
+	"github.com/envoyproxy/go-control-plane/envoy/api/v2/listener"
+	"github.com/envoyproxy/go-control-plane/envoy/api/v2/route"
+	bootstrap "github.com/envoyproxy/go-control-plane/envoy/config/bootstrap/v2"
+	accesslog "github.com/envoyproxy/go-control-plane/envoy/config/filter/accesslog/v2"
+	hcm "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/http_connection_manager/v2"
+	trace "github.com/envoyproxy/go-control-plane/envoy/config/trace/v2"
+	"github.com/envoyproxy/go-control-plane/pkg/util"
+	"github.com/ghodss/yaml"
+	"github.com/gogo/protobuf/jsonpb"
+	"github.com/gogo/protobuf/proto"
+	"github.com/gogo/protobuf/types"
+
+	meshconfig "istio.io/api/mesh/v1alpha1"
+	mixerv1 "istio.io/api/mixer/v1"
+	mixerclient "istio.io/api/mixer/v1/config/client"
+)
+
+// Default constants
+const (
+	ZipkinCluster    = "zipkin"
+	TelemetryCluster = "mixer_report_server"
+)
+
+var (
+	circuitBreakers = &cluster.CircuitBreakers{
+		Thresholds: []*cluster.CircuitBreakers_Thresholds{{
```

Comment:
Is there a reason we use this rather than using to Envoy's default CB (i.e. not providing a value at all)?

Reply:
Default CB is too restrictive. 1024 max is below what we expect.